src/udf/attributes_class.c

```
@@ -102,6 +103,38 @@ static int js_attributes_get_property
 	return 1 ;
 }
 
+// attribute-set to json
+static JSValue js_attributes_to_json
+(
+	JSContext *ctx,
+	JSValueConst this_val,
+	int argc,
+	JSValueConst *argv
+) {
```

Cppcheck (2.19.0)

[error] Common realloc mistake

(memleakOnRealloc)